velox/benchmarks/tpch/TpchBenchmark.cpp

```diff
@@ -165,8 +166,25 @@ int main(int argc, char** argv) {
     folly::runBenchmarks();
   } else {
     const auto queryPlan = queryBuilder->getQueryPlan(FLAGS_run_query_verbose);
-    const auto task = benchmark.run(queryPlan);
+    const auto [cursor, actualResults] = benchmark.run(queryPlan);
+    auto task = cursor->task();
     ensureTaskCompletion(task.get());
+    if (FLAGS_include_results) {
+      std::cout << "Results:" << std::endl;
+      bool printType = false;
+      for (const auto& vector : actualResults) {
+        // Print RowType only once.
+        if (!printType) {
```

Comment:
nit: printType sounds like if true, then we should print type. Consider changing the initial value and this check fr clarity.